kikaha-maven-plugin/source/kikaha/mojo/packager/ZipFileWriter.java

```diff
@@ -73,13 +74,23 @@ public void close() throws MojoExecutionException {
 
 	public void stripPrefix( final String... prefixes ) {
 		for ( final String prefix : prefixes )
-			prefixesToStripOutFromName.add( fixWindowsFileSeparator(prefix) );
+			prefixesToStripOutFromName.add( sanitizePath(prefix) );
 	}
 
-	private String fixWindowsFileSeparator(String path) {
-		if ( IS_WINDOWS_FILE_SEPARATOR && path.contains(WINDOWS_FILE_SEPARATOR) ) {
-			path = path.replace( WINDOWS_FILE_SEPARATOR, "/" );
+	private String sanitizePath(final String path) {
+		if(isWindowsPath(path)) {
+			return sanitizePathToWindows(path);
+		} else {
+			return path;
 		}
-		return path;
 	}
+
+	private boolean isWindowsPath(final String path) {
```

Comment:
Please consider using `System.getProperty("os.name")` to check which OS is being used.

Reply:
I considered using `System.getProperty("os.name")`, but if you are in a Linux environment and you are informed a Windows path, it will be a problem

Reply:
In JVM world we usually just use a UNIX-like path. When needed by the API that comes along with JDK (namely Path, File, etc), it is correctly converted. The problem lies on user input. User's should be encouraged to use UNIX-like path. On the few cases where a Windows path is required, it should run exclusively on Windows.

Reply:
Yes, I agree but when I came across this test, I saw that he was testing with Windows path and it was failing when running in a Linux environment. So this test actually does nothing and it could be removed, right?